### PR TITLE
Footer no longer overlaps content, fixes #76

### LINKED ISSUE #76: Looking for speakers section obscured on mobile
See screenshot

phph-responsive-issue

### PR DESCRIPTION
**Pull request by zaccolley**
footer isnt absolutely positioned anymore, has a box-shadow to flow down if the footer isnt at the bottom of the page

tried to remove the weird whitespace issues but you can add `?w=1` to the end of the url to see just code chanegs

### CODE REVIEW

File: public/css/style.css [hunk 1]
```diff
@@ -56,7 +53,7 @@ a{ color: #ec7748; }
 header {
 	padding: 15px 10px;
 	overflow: auto; /* clearfix */
-	
+
```
Comment (lboynton) [inline public/css/style.css]: Begone nasty whitespace! :)

**Comment by asgrim** [comment]
@zaccolley this conflits, needs rebase please

**Comment by lboynton** [comment]
Hmm I thought I commented on this earlier today...

I was thinking we could use flexbox magic here to keep the footer at the bottom, and prevent having a really large footer on the contact page.

**Comment by asgrim** [comment]
@lboynton you did, but on the issue: https://github.com/phphants/phph-site/issues/76#issuecomment-184563313 ;)

**Comment by lboynton** [comment]
Hah. I thought I was going crazy.

**Comment by asgrim** [comment]
Branch is very out of date. Closing this unless it's going to get updated (note: very soon we have major update in #96, so this would need to be re-worked into that)

**Pull request closed without merging**